schemas/payments_schema.py

```
@@ -50,28 +52,31 @@ def set_id(cls,values):
-    numberOfstars:int
+    numberOfstars:Optional[int]=None
+    durationDays:Optional[int]=None
     description:str
     
     
 class PaymentLink(BaseModel):
     bundle_id:str
     
 class ChapterPayment(BaseModel):
-    bundle_id:str
+    bundle_id:Optional[str]=None
```

Ruff (0.14.10)

`Optional` may be undefined, or defined from star imports

(F405)